e2e/tests/omnibus-base-goerli-andromeda.toml/Perps_Trading.e2e.js

```diff
@@ -0,0 +1,362 @@
+  });
+
+  it('should open a 0.1 btc position', async () => {
+    const btcMarketId = 200;
+    const marketId = btcMarketId;
+    const strategyId = 0;
+    const settlementStrategyId = extras.btc_pyth_settlement_strategy;
+    const strategy = await PerpsMarketProxy.getSettlementStrategy(marketId, strategyId);
+
+    const { feedId: priceFeedId, priceVerificationContract, commitmentPriceDelay } = strategy;
+
+    const price = await getPrice(priceFeedId);
+    const pythPrice = price.getPriceAsNumberUnchecked();
+    const sizeDelta = ethers.utils.parseEther('0.1');
+
+    await setSettlementDelay({ strategyId, marketId, delay: 1 });
+    const tx = await PerpsMarketProxy.connect(wallet)
+      .commitOrder({
+        marketId,
+        accountId,
+        sizeDelta,
+        settlementStrategyId,
+        acceptablePrice: ethers.utils.parseEther(Math.floor(pythPrice * 2).toString()),
+        referrer: ethers.constants.AddressZero,
+        trackingCode: ethers.constants.HashZero,
+      })
+      .catch(parseError);
+    const commitReceipt = await tx.wait().catch(parseError);
+    log('Order committed');
+
+    const block = await provider.getBlock(commitReceipt.blockNumber);
+    const commitmentTime = block.timestamp;
+
+    // wait 1 seconds for settlement delay
+    await wait(1000);
+
+    await fulfillOracleQuery({
+      wallet,
+      oracleFeedId: priceFeedId,
+      priceVerificationContractAddress: priceVerificationContract,
+      timestamp: commitmentTime + commitmentPriceDelay.toNumber(),
+    });
+
+    const settleTx = await PerpsMarketProxy.connect(wallet)
+      .settleOrder(accountId)
+      .catch(parseError(error));
+    await settleTx.wait().catch(parseError(error));
+    log('Open order settled');
+
+    const position = await PerpsMarketProxy.getOpenPosition(accountId, marketId);
+    assert.equal(parseFloat(ethers.utils.formatUnits(position.positionSize)), 0.1);
+  });
+
+  it('should close a 0.1 btc position', async () => {
+    const btcMarketId = 200;
+    const marketId = btcMarketId;
+    const strategyId = 0;
+
+    const settlementStrategyId = extras.btc_pyth_settlement_strategy;
+    const strategy = await PerpsMarketProxy.getSettlementStrategy(marketId, strategyId);
+
+    const { feedId: priceFeedId, priceVerificationContract, commitmentPriceDelay } = strategy;
+    const price = await getPrice(priceFeedId);
+    const pythPrice = price.getPriceAsNumberUnchecked();
+    const sizeDelta = ethers.utils.parseEther('-0.1');
+
+    const tx = await PerpsMarketProxy.connect(wallet)
+      .commitOrder({
+        marketId,
+        accountId,
+        sizeDelta,
+        settlementStrategyId,
+        acceptablePrice: ethers.utils.parseEther(Math.floor(pythPrice * 0.5).toString()),
+        referrer: ethers.constants.AddressZero,
+        trackingCode: ethers.constants.HashZero,
+      })
+      .catch(parseError);
+    const commitReceipt = await tx.wait().catch(parseError);
+    log('Close order committed');
```

Comment:
Would love to see more logs with all the parameters flying around. But that's not a blocker

Reply:
Added some more logs